tasklist/config/docker-compose.yml

```
@@ -442,3 +441,35 @@ services:
     networks:
       - zeebe_network
     restart: always
+
+  optimize_identity:
```

Just missed this anchor tag
```suggestion
  optimize_identity: &optimize_identity
```